.changeset/mighty-cars-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-svelte': major
```

Comment:
This only changes the interactive command line prompt, not the API, so to me this doesn't need to be a major
```suggestion
'create-svelte': minor
```